agent/bench-scripts/pbench-uperf

```diff
@@ -425,36 +426,39 @@ if [ ! -z "$clients" -a "$postprocess_only" != "y" ]; then
 		error_log "    vsock-servers(${v_cnt}): ${vsock_servers}"
 		exit 1
 	fi
-fi
-# before we run a test, verify clients and servers are accessible and install
-# uperf if necessary
-if [ "$postprocess_only" != "y" ]; then
-	# Always install locally
-	# FIXME: why is this necessary?
-	check_install_rpm "${benchmark_rpm}" "${ver}" "${match}"
+
+	if [[ ${check_local} -gt 0 ]]; then
+		check_install_rpm "${benchmark_rpm}" "${ver}" "${match}"
```

Comment:
This results in only checking for the installed RPM on the local host when we are actually going to use the local host as a client or server.

Reply:
That sounds great, but if it was obscure and important enough to warrant a comment here, perhaps it should (have) had one in the code?